docker/build

```diff
@@ -0,0 +1,47 @@
+# Copyright 2020 The OpenEBS Authors. All rights reserved.
```

Comment:
This file needs shell linting to be performed as it's being ignored as it doesn't have the file extension the workflow is looking for.